src/workspace-data/Data.js

```diff
@@ -748,13 +766,6 @@ export const WorkspaceData = _.flow(
 
     const toSortedPairs = _.flow(_.toPairs, _.sortBy(_.first));
 
-    const deleteColumnUpdateMetadata = ({ attributeName, entityType }) => {
```

Comment:
I inlined this in `DataTable.js` rather than drilling it down, it seemed to me like DataTable had a reference to `entityMetadata` and `setEntityMetadata` already that I could use.